R-package/DESCRIPTION

```diff
@@ -26,7 +26,8 @@ Authors@R: c(
     person("Daniel", "Lemire", role = c("ctb")),
     person("Victor", "Zverovich", role = c("cph")),
     person("IBM Corporation", role = c("ctb")),
-    person("David", "Cortes", role = c("ctb"))
+    person("David", "Cortes", role = c("ctb")),
+    person("Michael", "Mayer", role = c("ctb"))
```

Comment:
@mayer79 if you'd like to change anything about this (like add an email address or a title like 'Dr.'), let us know